lib/dialects/mssql/query-generator.js

```diff
@@ -680,6 +680,25 @@ var QueryGenerator = {
     return sql;
   },
 
+  getPrimaryKeyConstraint(table, attributeName) {
+    const tableName = wrapSingleQuote(table.tableName || table);
+    const sql = [
```

Comment:
@sushantdhiman I wanted to keep the query readable yet copy pastable when logged so went for this. Using backticks keeps the same formatting as defined when logged.